ocs_ci/ocs/platform_nodes.py

```diff
@@ -526,6 +529,10 @@ def restart_nodes(self, nodes, timeout=300, wait=True):
             )
 
         self.aws.restart_ec2_instances(instances=instances)
+    
+        logger.info(f"Waiting for 30 seconds...")
+        time.sleep(30)
```

Comment:
Do not add extra wait time if wait=False

Reply:
Done